src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java

```diff
@@ -216,6 +300,7 @@ public KNNQueryBuilder(StreamInput in) throws IOException {
             vector = in.readFloatArray();
             k = in.readInt();
             filter = in.readOptionalNamedWriteable(QueryBuilder.class);
+            efSearch = in.readOptionalInt();
```

Comment:
For version compatibility for multiple nodes in difference versions OpenSearch cluster, we need version check to determine if it need read and write efSearch value. Example:
```
if (isClusterOnOrAfterMinRequiredVersion(KNNConstants.EF_SEARCH_KEY)) {
                efSearch = in.readOptionalFloat();
            }
```

Reply:
Got it. I saw that part and wasn't sure why it was required. Thanks for explanation, I will change it